spec/support/shared/contexts/msf/simple/framework/modules/loading.rb

```diff
@@ -154,6 +154,7 @@ def load_and_create_module(options={})
     )
 
     module_set = module_set_for_type(module_type)
+    module_set.recalculate
```

Comment:
I wonder if there's a way of performing this action once at a higher level within the test setup? Otherwise, if I understand correctly, we'll be adding an extra 1-2 seconds per payload test which calls `load_and_create_module`? 👀